db/schema.rb

```diff
@@ -29,6 +29,10 @@
     t.datetime "reject_by_default_at"
     t.boolean "rejected_by_default", default: false, null: false
     t.integer "reject_by_default_days"
+    t.datetime "decline_by_default_at"
```

Comment:
I wonder if `decline_by_default_at` really belongs on `ApplicationChoice`. If you [look at the design](https://apply-mvp-prototype.herokuapp.com/applications?phase=dbd&state=has-decisions), it shows "You have 12 days (until 9 December 2019) to respond to any offers". Displaying that data will mean that you have to get `decline_by_default_at` from one of the choices, which feels a bit clunky.

Reply:
That's how I felt about this originally, but:
- states and state transitions are all ApplicationChoice-specific already
- and there will be a sweeper task performing the actual 'decline' part of DBD, operating at the ApplicationChoice level (e.g. marking each choice as declined, possibly sending a notification to the provider)

Reply:
> states and state transitions are all ApplicationChoice-specific already

Having the dates on ApplicationChoice is already causing problems, like we see in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/738/files#r351666170. Are we going to have the same problem here?

See also https://trello.com/c/zPsXM1TW.

Reply:
I'd be happy to merge this as-is for now — maybe we should take a step back and review the way we organise this information in the near future? It also came up in #738.

Reply:
Missed your comment @tijmenb 😄

Reply:
Yeah, good idea.